packages/plugin/src/theia.d.ts

```diff
@@ -4453,6 +4453,133 @@ declare module '@theia/plugin' {
         provideDocumentSymbols(document: TextDocument, token: CancellationToken): ProviderResult<SymbolInformation[] | DocumentSymbol[]>;
     }
 
+    /**
+    * Represents a color in RGBA space.
+    */
+    export class Color {
+
+        /**
+        * The red component of this color in the range [0-1].
+        */
+        readonly red: number;
+
+        /**
+        * The green component of this color in the range [0-1].
+        */
+        readonly green: number;
+
+        /**
+        * The blue component of this color in the range [0-1].
+        */
+        readonly blue: number;
+
+        /**
+        * The alpha component of this color in the range [0-1].
+        */
+        readonly alpha: number;
+
+        /**
+        * Creates a new color instance.
+        *
+        * @param red The red component.
+        * @param green The green component.
+        * @param blue The blue component.
+        * @param alpha The alpha component.
+        */
+        constructor(red: number, green: number, blue: number, alpha: number);
+    }
+
+    /**
+    * Represents a color range from a document.
+    */
+    export class ColorInformation {
+
+        /**
+        * The range in the document where this color appears.
+        */
+        range: Range;
+
+        /**
+        * The actual color value for this color range.
+        */
+        color: Color;
+
+        /**
+        * Creates a new color range.
+        *
+        * @param range The range the color appears in. Must not be empty.
+        * @param color The value of the color.
+        * @param format The format in which this color is currently formatted.
```

Comment:
@akurinnoy You defined format, but constructor doesn't contains this param.

Reply:
thanks, fixed